_data/developer.yml

```
@@ -101,6 +101,13 @@ websites:
     img: bitnami.png
     twitter: bitnami
 
+  - name: bitrise
```

```suggestion
  - name: Bitrise
```
Their webpage and social media uses this capitalisation